bower.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "remotestorage-modules",
+  "description": "remotestorage modules collection",
+  "keywords": "remotestorage modules collection",
+  "main": "src/*.js",
+  "ignore": [
+  ],
+  "version": "0.0.1",
+  "private": false,
+  "author": "Niklas Cathor <nilclass@riseup.net>",
+  "license": "AGPL",
```

Comment:
All rs.js projects are licensed under MIT these days. We removed the MIT/GPL double licensing 2 or so years ago in favor of the more permissive one, as it is compatible with GPL anyway.